UHResidentInformationAPI/V1/Infrastructure/EmailAddress.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UHResidentInformationAPI.V1.Infrastructure
+{
+    [Table("CCEmail")]
+    public class EmailAddresses
+    {
+       [Column("ContactNo")]
+        [MaxLength(9)]
+        [Key]
+        public int ContactID { get; set; }
+
+        [Column("Email")]
+        [MaxLength(9)]
+        public int EmailAddress { get; set; } 
+
+        [Column("EmailType")]
+        [MaxLength(9)]
+        public int EmailType { get; set; } 
+
+        [Column("ModDate")]
```

Comment:
I could be wrong but I think the columns mentioned here need to be written the same as the column in the table. This is called `modDate` in the schema.sql file.